Podfile

```diff
@@ -4,5 +4,6 @@ use_frameworks!
 pod 'NBNRequestKit', :git => 'https://github.com/1ec5/RequestKit.git', :branch => 'mapbox-podspec'
 
 target 'MapboxDirectionsTests' do
+  pod 'Polyline', '~> 3.0'
```

Comment:
This dependency needs to be added to the implicit target above, like NBNRequestKit. This section is for dependencies specific to the test bundle.